blivet/dbus/blivet.py

```diff
@@ -208,3 +208,8 @@ def InitializeDisk(self, object_path):
         self.RemoveDevice(object_path)
         device = self._get_device_by_object_path(object_path)
         self._blivet.initialize_disk(device)
+
+    @dbus.service.method(dbus_interface=BLIVET_INTERFACE)
+    def Commit(self):
+        """ Commit pending changes to disk. """
+        self._blivet.do_it()
```

Comment:
I know this is a bit preliminary, but shouldn't we catch exceptions here and transform them into DBus exceptions?